File: dotnet/niuz.application/events/ContractSigned.cs
namespace niuz.application.events
{
    public class ContractSigned
    {
        public ContractSigned(string authorId, string authorName, string contractType, int rate, string bankAccount)
        {
            AuthorId = authorId;
            AuthorName = authorName;
            ContractType = contractType;
            Rate = rate;
            BankAccount = bankAccount;
        }

        public string AuthorId { get; }
        public string AuthorName { get; }
        public string ContractType { get; }
        public int Rate { get; }
        public string BankAccount { get; }
    }
}

File: java/src/main/java/com/dddeurope/niuz/events/ContractSigned.java
package com.dddeurope.niuz.events;

public class ContractSigned {
    private String authorId;
    private String contractType;
    private Integer rate;
    private String bankAccount;
    private String authorName;

    public ContractSigned(String authorId, String contractType, Integer rate, String bankAccount, String authorName) {
        this.authorId = authorId;
        this.contractType = contractType;
        this.rate = rate;
        this.bankAccount = bankAccount;
        this.authorName = authorName;
    }

    public String getAuthorId() {
        return authorId;
    }

    public String getContractType() {
        return contractType;
    }

    public Integer getRate() {
        return rate;
    }

    public String getBankAccount() {
        return bankAccount;
    }

    public String getAuthorName() {
        return authorName;
    }
}
